pages/common/zm.md

```diff
@@ -0,0 +1,16 @@
+# zm
+
+> A tool for managing articles of newspapers and blogs.
+> More information: <https://github.com/ZERMZeitung/zm2>.
+
+- Make a new draft:
+
+`zm new`
+
+- Edit a draft:
+
+`zm edit`
+
+- Publish a draft:
+
+`zm publish`
```

Comment:
I've read `zm` shell script, `zm publish` means commit a draft to git right? Mention git commit here is clearer.